src/katsdpcontroller/generator.py

```diff
@@ -798,6 +798,9 @@ def _make_fgpu(
         fgpu.pass_telstate = False
         fgpu.data_suspect_sensor = data_suspect_sensor
         fgpu.data_suspect_range = (2 * i, 2 * i + 2)
+        # Identify the antenna, if the input labels are consistent
+        if input_labels[0][:-1] == input_labels[1][:-1]:
```

Comment:
Is this just a sanity-check to see whether both pols are coming from the same antenna?

Reply:
Yes. If an engine is being fed by different antennas (or someone is using it with sources that don't follow the MK input naming convention), avoid producing misleading antenna information.